ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,42 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+from hypothesis import strategies as st
+
+
+# asanyarray
+@handle_frontend_test(
+    fn_tree="numpy.asanyarray",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+    dtype=helpers.get_dtypes("valid", full=False),
+    order=st.sampled_from(["C", "F", "A", "K"]),
+    like=helpers.dtype_and_values(num_arrays=1),
+)
+def test_numpy_asanyarray(
+    dtype_and_x,
+    dtype,
+    order,
+    *,
+    like,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    dtype, x = dtype_and_x
```

Comment:
You need to do this for `like` as well as currently `like` bring input to the the `test_frontend_function` is a tuple `(dtype, values)` which will throw an error. Are you able to run this test locally to be able to check if it's running and what errors it's throwing yourself? It isn't necessary to request my review on every incremental change if you know it is still failing. You can try to formulate a question to me regarding the error you are seeing locally. This may speed up the process 😊

Reply:
I'm not able to run the test locally because it throws errors from other functions. Thanks and I'm sorry for disturbing you with review. I won't do that again.

Reply:
@Taciturny You should be able to run this test on its own locally - see this [guide](https://lets-unify.ai/ivy/contributing/setting_up.html#setting-up-testing) to setting up local testing 😊

Reply:
I know how to do it. But there are a lot of bugs ma.

Reply:
Ah that's good, so you can go through the error stack trace and try to figure out what the issues are. Googling the some keywords in the error stack trace is always the first step to getting to the bottom of the issue :)

Reply:
I tried it. But had error because of bugs from other functions. I'm thinking of dropping out. Thank you for your help all these 3 months.

Reply:
Look at this. This is similar to my issue. [https://github.com/Taciturny/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_from_existing_data.py#:~:text=)-,%23%20asarray,-%40handle_frontend_test(](url). Thanks